app/presenters/contacts_finder_presenter.rb

```diff
@@ -1,105 +0,0 @@
-class ContactsFinderPresenter
```

Comment:
I checked the HMRC contact index page on dev and everything is still working as expected. This ContactsFinderPresenter was only ever called inside the [`publish_finder`](https://github.com/alphagov/contacts-admin/blob/master/app/models/contact_group.rb#L38-L41) method in contact_group.rb, which in turn was commented out.